app/routes/entries/index.tsx

```diff
@@ -12,7 +12,7 @@ export default function EntryIndexPage() {
           <div key={letter}>
             <Link
               className="text-xl font-bold text-red-600 hover:text-red-400"
-              to={`/entries/browse/${letter}`}
+              to={`/entries/browse/${letter}/1`}
```

Comment:
Instead of hardcoding the `1` on the end of the path here, let's just link to `/entries/browse/${letter}`, and rely on the route loader at that path to redirect us as required. It just makes the code (slightly) cleaner by not having to put the page number in multiple places, and easier to change later if we were to change the page numbering approach.

Reply:
Done